.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,25 @@
+name: dsp-appsec-trivy
+on:
+  schedule:
+    - cron:  '0 14 * * 1' # each Monday at 9am EST
+
+jobs:
+  appsec-trivy:
+    # Parse Dockerfile and build, scan image if a "blessed" base image is not used
```

Comment:
This looks like incorrect copy/paste? It actually validates the existing docker image

Reply:
Could we add the intention and mechanism of this check in a comment, and why we do it on schedule and NOT PR?

Reply:
I think this looks correct - it checks if the _base_ image in the Dockerfile is "blessed", and if not (as in your case) then it will scan your image instead (otherwise, it would skip the scan if blessed).

Reply:
And feel free to add any clarifying comments there, of course!

Reply:
Sounds good